scripts/tools/tools-object.js

```diff
@@ -33,28 +34,40 @@ const createToolObject = (toolFile, repositoryUrl, isAsyncAPIrepo) => {
 
 async function convertTools(data) {
   let appendData = {};
+  const dataArray = data.items;
+
   for(var index in categoryList){
     appendData[categoryList[index].name] = {
       description: categoryList[index].description,
       toolsList: []
     };
   }
-  const dataArray = data.items;
+
   for (let tool of dataArray) {
+
```

Comment:
I think that code inside loop should be also wrapped by `try catch`, only to console log possible errors in code.

Reply:
I don't think it is needed. In case of errors other than validation we should throw error and not console log. And this is happening. In the end conversion function is wrapped in try/catch in place where it is called

Reply:
> In the end conversion function is wrapped in try/catch in place where it is called

Where? I don't see any `try/catch`. I only see some checking error by `validate` function, but it's not a typical try/catch.

Reply:
Yeah, I agree with @magicmatatjahu, we should have `try/catch` block here and I got some errors on this part of code only. Basically, we aren't putting any error handling for axios function, so it would be better if we make a try/catch block there to get correct error log messages.

Reply:
> Where? I don't see any try/catch. I only see checking errors by validate function, but it's not a typical try/catch.

I think @derberg is talking about the `try/catch` block used in `scripts/build-tools.js` file which applies to every function call made inside the functionality.

Reply:
It can be fixed in other PR, I don't want to block you guys :)